toolchain/driver/driver.cpp

```diff
@@ -32,24 +32,24 @@
 
 namespace Carbon {
 
-auto Driver::FindPreludeFiles(llvm::StringRef data_dir,
+auto Driver::FindPreludeFiles(llvm::StringRef core_package_dir,
                               llvm::raw_ostream& error_stream)
     -> llvm::SmallVector<std::string> {
   llvm::SmallVector<std::string> result;
 
   // Include <data>/core/prelude.carbon, which is the entry point into the
   // prelude.
   {
-    llvm::SmallString<256> prelude_file(data_dir);
+    llvm::SmallString<256> prelude_file(core_package_dir);
     llvm::sys::path::append(prelude_file, llvm::sys::path::Style::posix,
-                            "core/prelude.carbon");
+                            "prelude.carbon");
     result.push_back(prelude_file.str().str());
   }
 
   // Glob for <data>/core/prelude/**/*.carbon and add all the files we find.
-  llvm::SmallString<256> prelude_dir(data_dir);
+  llvm::SmallString<256> prelude_dir(core_package_dir);
```

Comment:
Why `core_package_dir` in some spots but `core_package_path` in others? What's the difference between these values?

Reply:
No difference. I have a mild preference for `_path` when it's actually a path string, and `_dir` for something more like a directory handle. But this code used `_dir` for `prelude_dir` so I tried to stay consistent with that in this file.

I'm happy to consolidate these names in either direction in a follow-up, can discuss which names folks would prefer because my preference is very mild here.